src/Features/Core/Portable/Copilot/IProposalAdjusterService.cs

```diff
@@ -102,26 +109,40 @@ private async Task<ProposalAdjustmentResult> TryAdjustProposalInCurrentProcessAs
         var forkedRoot = await forkedDocument.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
can we move this into TryGetAddImportTextChangesAsync

it can be a helper, then it can be used as part of TryGetFormattingTextChangesAsync as well.  

basically, i'm having a lot of trouble understanding teh flow, and which steps update data and how that flows intot he next.

Reply:
OK, I've reworked it to be much simpler. Each fixer just modifies the document in order as it goes down the line and the changes are extracted at the end.